run_tests.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+import unittest
+
+loader = unittest.TestLoader()
+tests = loader.discover(start_dir='.', pattern='*_test.py')
```

Comment:
I think this can be done with command line flags. I'm not sure of best practices.